tensorflow/lite/kernels/pooling_test.cc

```diff
@@ -26,8 +26,22 @@ using ::testing::ElementsAreArray;
 
 class BasePoolingOpModel : public SingleOpModel {
  public:
-  // TODO(ahentz): Also test different activation types, bias, padding types,
-  // stride values.
+  BasePoolingOpModel(const BuiltinOperator& type, const TensorData& input,
+                     int filter_width, int filter_height,
+                     const Padding& padding,
+                     const ActivationFunctionType& fused_activation_function,
+                     const TensorData& output) {
+    input_ = AddInput(input);
+    output_ = AddOutput(output);
+
+    SetBuiltinOp(type, BuiltinOptions_Pool2DOptions,
+                 CreatePool2DOptions(builder_, padding, filter_width, filter_height, filter_width,
```

Comment:
a little typo here: the first filter_width and filter_hight should be stride_width, and stride_height.

Reply:
@haozha111 , sorry for the overlook, i have fixed the issue, kindly check and approve.